library/crates/remote/BUILD.better_any-0.1.1.bazel

```diff
@@ -0,0 +1,62 @@
+"""
```

Comment:
why do we have so many generated Bazel files? what triggers the auto-creation of these auto-generated files @vmax ?